### PR TITLE
Miscellaneous changes

### PR DESCRIPTION
**Pull request by wkaluza**

### CODE REVIEW

File: libs/core/src/byte_array/const_byte_array.cpp [hunk 1]
```diff
@@ -24,12 +24,12 @@ namespace byte_array {
 
 ConstByteArray ConstByteArray::ToBase64() const
 {
-  return ::fetch::byte_array::ToBase64(*this);
+  return fetch::byte_array::ToBase64(*this);
```
Comment (ejfitzgerald) [inline libs/core/src/byte_array/const_byte_array.cpp:27]: Don’t need namespaces here at all

**Review by ejfitzgerald — COMMENTED** [review]
Seems legit. Worth getting @baykaner to have a look through too. One minor pedantic comment

File: libs/core/src/byte_array/const_byte_array.cpp (hunk 1 above)
Comment (wkaluza) [reply libs/core/src/byte_array/const_byte_array.cpp:27]: I thought so too, but we do need them here - at least on Clang 6 without namespace qualification this call resolves to `ConstByteArray::ToBase64` and fails with `too many arguments to function call, expected 0, have 1`.

Comment (ejfitzgerald) [reply libs/core/src/byte_array/const_byte_array.cpp:27]: Of course you are correct! 😁

**Review by baykaner — APPROVED** [review]

File: libs/core/src/byte_array/const_byte_array.cpp (hunk 1 above)
Comment (wkaluza) [reply libs/core/src/byte_array/const_byte_array.cpp:27]: Why can't the compiler just *guess* which function I mean? Stupid compiler.

**Pull request merged**